src/modules/commands/fight.js

```diff
@@ -0,0 +1,22 @@
+const isEmpty = require('lodash/isEmpty');
+const random = require('lodash/sample');
+const command = require('../command.filter');
+
+const fights = async function (request, context) {
+    const { i18n, send } = context;
+    const { args: opponent } = request;
+
+    if (isEmpty(opponent)) {
+        send(i18n('fight.userisnotdefined'));
+
+        return request;
+    }
+    const fighters = ['you', opponent];
+    const winner = random(fighters);
+    const loser = random(fighters);
+
+    send(i18n('fight.wins', { win: { winner }, lose: { loser } }));
```

Comment:
что это за конструкция? что ты тут делаешь? в обьекты вкладываешь в обьекты... все проще же

```suggestion
    send(i18n('fight.wins', { winner, loser }));
```

и смотри выше мы уже положили в winner/looser именно discord id пользователей! Так что никаких больше преобразований